lakefile.lean

```diff
@@ -43,6 +43,11 @@ lean_exe checkYaml where
   root := `scripts.checkYaml
   supportInterpreter := true
 
+/-- `lake exe declsIn` lists all declarations provided in the given files. -/
```

Comment:
Could you add instructions here, that users will want to use `scripts/decl-name-changes.sh`?